pkg/sass-parser/lib/src/statement/content-rule.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright 2025 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import * as postcss from 'postcss';
+import type {AtRuleRaws} from 'postcss/lib/at-rule';
+
+import {ArgumentList, ArgumentListProps} from '../argument-list';
+import {LazySource} from '../lazy-source';
+import {NodeProps} from '../node';
+import * as sassInternal from '../sass-internal';
+import * as utils from '../utils';
+import {Statement, StatementWithChildren} from '.';
+import {_AtRule} from './at-rule-internal';
+import {interceptIsClean} from './intercept-is-clean';
+import * as sassParser from '../..';
+
+/**
+ * The set of raws supported by {@link ContentRule}.
+ *
+ * @category Statement
+ */
+export interface ContentRuleRaws extends Omit<AtRuleRaws, 'params'> {
+  /**
+   * Whether to content an empty argument list. If the argument list isn't
```

Comment:
```suggestion
   * Whether to include an empty argument list. If the argument list isn't
```
I was confused by the comment, but IIUC this was unintentionally search-replaced

Reply:
That is accurate :smile: